samples/matter/light_switch/src/app_config.h

```diff
@@ -24,3 +24,6 @@
 
 #define SYSTEM_STATE_LED DK_LED1
 #define IDENTIFY_LED DK_LED2
+
+#define FACTORY_RESET_SIGNAL_LED DK_LED3
```

Comment:
Maybe it would be good to add #if NUMBER_OF_LEDS == 4? I know it doesn't change anything in functionality, but just to be clear.